### ISSUE TITLE
Errors for `{types| fields}-have-descriptions` rules are reported twice

### ISSUE THREAD

**Issue by loicplaire** [cjoudrey/graphql-schema-linter#132]
It seems that the errors for `types-have-descriptions` and `fields-have-descriptions` rules are reported twice e.g:

```bash
/git/graphql-schema-linter-duplicated-errors/schema.graphql
1:1 The object type `Challenge` is missing a description.        types-have-descriptions
1:1 The object type `Challenge` is missing a description.        types-have-descriptions
2:3 The field `Challenge.id` is missing a description.           fields-have-descriptions
2:3 The field `Challenge.id` is missing a description.           fields-have-descriptions
```
I setup a quick repo with an example: https://github.com/loicplaire/graphql-schema-linter-duplicated-errors

Am I missing something?

**Comment by cjoudrey** [comment]
Thanks for the report Loic! That is strange indeed. I will look soon.

**Comment by cjoudrey** [comment]
This should be fixed in `0.1.6`. Sorry for the inconvenience.

**Comment by loicplaire** [comment]
Thanks for the quick fix @cjoudrey! 👍
